annotation/src/main/java/org/cbioportal/genome_nexus/annotation/web/AnnotationController.java

```diff
@@ -218,6 +225,47 @@ public AnnotationController(VariantAnnotationService variantAnnotationService,
         return getHotspotAnnotation(variants);
     }
 
+    @ApiOperation(value = "Retrieves mutation assessor information for the provided list of variants",
+        nickname = "getMutationAssessorAnnotation")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = "Success",
+            response = MutationAssessor.class,
+            responseContainer = "List"),
+        @ApiResponse(code = 400, message = "Bad Request")
+    })
+    @RequestMapping(value = "/mutation_assessor/{variants:.+}",
+        method = RequestMethod.GET,
+        produces = "application/json")
+    public List<MutationAssessor> getMutationAssessorAnnotation(
+        @PathVariable
+        @ApiParam(value="Comma separated list of variants. For example 7:g.140453136A>T,12:g.25398285C>A",
+            required = true,
+            allowMultiple = true)
+            List<String> variants)
+    {
+        List<MutationAssessor> mutationAssessors = new ArrayList<>();
+        for (String variant : variants)
+        {
+            mutationAssessors.add(getMutationAnnotation(variant));
+        }
+        return mutationAssessors;
+    }
+
+    @ApiOperation(value = "Retrieves mutation assessor information for provided list of variants",
+        nickname = "postMutationAssessorAnnotation")
+    @RequestMapping(value = "/mutation_assessor",
+        method = RequestMethod.POST,
+        produces = "application/json")
+    public List<MutationAssessor> postMutationAssessorAnnotation(
+        @RequestParam
+        @ApiParam(value="Comma separated list of variants. For example 7:g.140453136A>T,12:g.25398285C>A",
+            required = true,
+            allowMultiple = true)
+        List<String> variants)
```

Comment:
A general concern: What we have been doing so far with the `list parameters` is not optimal. We should try to avoid comma separated query parameters, and replace all existing ones with body parameters. We don't need to worry about it for now though, it is already like this everywhere :)